python/cudf/cudf/core/tools/datetimes.py

```diff
@@ -652,3 +653,23 @@ def _isin_datetimelike(
 
     res = lhs._obtain_isin_result(rhs)
     return res
+
+
+def to_iso_calendar(arg):
```

Comment:
This seems to be an internal API, either document it or make it an internal API?

Reply:
Sounds good Prem! I'll make this an internal API!